Simulation/TSoG_X1_Sim.m

```diff
@@ -3,7 +3,7 @@
 % or without an input Test Case (the sim will run with default inputs if no Test
 % Case is provided)
```

Comment:
Need to add details on how to use the sim with the new input 'Plane' structure:
- Has to be second
- Will use default if not provided